sbin/common/config_init.sh

```diff
@@ -371,10 +371,20 @@ function configDefaults() {
   local unameSys=$(uname -s)
   local unameOSSysVer=$(uname -sr)
   if [ "${unameSys}" == "Linux" ]; then
-    # Linux distribs add more useful distrib in the single line file /etc/system-release
+    # Linux distribs add more useful distrib in the single line file /etc/system-release,
+    # or property file /etc/os-release
     if [ -f "/etc/system-release" ]; then
       local linuxName=$(tr -d '"' < /etc/system-release)
       unameOSSysVer="${unameOSSysVer} : ${linuxName}"
+    elif [ -f "/etc/os-release" ]; then
```

Comment:
Is it possible that an O/S would ever have both of these folders?

Reply:
Yes, some do, but they have the same os name value. most have system-release, some also have os-release and quite often both are symlinks to a vendor release file, eg."redhat-release"